File: 27.03.25/ps.js
// let num=60;

// let reverse=num.toString().split("").reverse().join("")

// if(reverse==num){
//     console.log("palindrome")
// }
// else{
//     console.log("not")
// }

// palindrome in range

// let arr=[22,44,32,34,33]

// for(let i=0;i<arr.length;i++){
//     let reverse=arr[i].toString().split("").reverse().join("");
//     if(reverse==arr[i]){
//         console.log(`${arr[i] } is palindrome`)
//     }
//     else{
//         console.log("no")
//     }
// }


function prime(num){
    if(num<=1) return false
    for(i=2;i<=Math.sqrt(num);i++){
        if(num%i==0){
            return false;
        }
    }
    return true
}


function check(arr){
    let empty=[]
    for( i of arr){
        if(prime(i)){
            empty.push(i)
        }
    }
    return empty
}
console.log(check([32,42,2]))

File: 27.03.25/ps.py
# palindrome 

num=90
# rever=str(num)[::-1]
# if rever==str(num):
#     print("yes")
# else:
#     print("no")
# print("yes" if str(num)==str(num)[::-1] else "no")

# arr=[32,3,2,33,22,33]
# for i in arr:
#     if str(i)==str(i)[::-1]:
#         print(f"{i} is palindrome")
#     else:
#         print("no")


#prime

def prime(num):
    if num <=1:
        return False
    for i in range(2,int(num**0.5)+1):
        if num%i==0:
            return False
    return True

def check(arr):
    em=[]
    for i in arr:
        if prime(i):
            em.append(i)
    return em
print(check({22,32,32,31}))
